source/server/winston-adapter.js

```diff
@@ -36,13 +36,11 @@ const WinstonAdapter = Space.Logger.Adapter.extend('Space.Logger.WinstonAdapter'
   }
 });
 
-WinstonAdapter.console = function(options) {
-  if (options == null) {
```

Comment:
I'm thinking we should not impose our own opinion on defaults, winston already has defaults